.github/workflows/e2e-test-darts-cifar10.yaml

```diff
@@ -9,9 +9,6 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
```

Comment:
Did we require GitHub Token before for our E2Es?

Reply:
Yes, the token is required for the `manusa/actions-setup-minikube`.